Program.cs

```diff
@@ -629,57 +654,93 @@ public void itemShop()
             if (hasPlayerBeenInShop == true)
             {
                 somethingCool();
+                if (playerHasWatch == false)
+                {
+                    GetWatch(totalXP);
+                }
                 shopCommonPlace();
             }
             else
             {   
-                TypeWriter.WriteLine($"Hello {cottonUserName} ");
                 shopCommonPlace();
                 hasPlayerBeenInShop = true;
             }
             
         }
 
-        private void shopCommonPlace()
+        private void GetWatch(int totalXP)
         {
-            TypeWriter.WriteLine(new Text("Are you looking for "),
-                                 new Text("(w)weapons, ", Colours.Attack, TypeWriter.Speed.Talk),
-                                 new Text("(a)armour ", Colours.Protection, TypeWriter.Speed.Talk),
-                                 new Text("or ", Colours.Speech, TypeWriter.Speed.Talk),
-                                 new Text("(m)medicine ", Colours.Health, TypeWriter.Speed.Talk),
-                                 new Text(":)", Colours.Speech, TypeWriter.Speed.Talk));
-            if (hasPlayerBeenInShop == true)
+            if (totalXP >= 300)
             {
-               if (playerHasSeenMorse == true)
-               {
-                   TypeWriter.WriteLine("(h) to hear it again or (l) to leave", TypeWriter.Speed.Talk);
-               }
-               else if (playerHasSeenMorse == false)
-               {
-                   TypeWriter.WriteLine("(h) to hear something cool or (l) to leave", TypeWriter.Speed.Talk);
-               }
+                TypeWriter.WriteLine($"Hey {cottonUserName} do you want to trade {priceForWatch} Gold for my pocket watch: yes/no", TypeWriter.Speed.Talk);
+                string Awnser = Console.ReadLine();
+                if (Awnser == "yes")
```

Comment:
uppercase will not be accepted.

Reply:
This will get painful I know... perhaps we have a wrapper method that we can use in place of Console.Readline() that will do a readline, and then lower the casing and return it .. so you never need to worry about it again :)